<?xml version="1.0" encoding="utf-8"?>
<launch>
  
  <!-- We resume the logic in empty_world.launch, changing the name of the world to be launched -->
  <include file="$(find sawyer_gazebo)/launch/sawyer_world.launch">
    <arg name="electric_gripper" value="true"/>
  </include>
    <node name="joint_trajectory_action_server" pkg="intera_interface" type="joint_trajectory_action_server.py" respawn="true" output="screen">
  </node>
  <include file="$(find sawyer_moveit_config)/launch/sawyer_moveit.launch">
    <arg name="electric_gripper" value="true"/>
  </include>

</launch>